deployment/cfn/vpc.py

```diff
@@ -47,11 +43,8 @@ class VPC(StackNode):
         'Region': 'us-east-1',
         'StackType': 'Staging',
         'KeyName': 'mmw-stg',
-        'IPAccess': '0.0.0.0',
+        'AvailabilityZones': ['us-east-1b', 'us-east-1d'],
```

Comment:
This is a stupid question, but why default to availability zones b/d instead of a/b?